.travis.yml

```diff
@@ -1,11 +1,20 @@
+language: shell
+
 # We use Docker → we need sudo.
 sudo: required
 
 services:
   - docker
 
 script:
-  - bash ci/build.sh
+  - ci/build.sh
```

Comment:
```console
ci/build.sh: Permission denied
```

You need to add execution permission to run it directly:

```console
chmod +x ci/build.sh
git commit -am 'Add executable permission.'
```

Reply:
Done, thanks!

Surprisingly all projects have _something_ different...

Reply:
This one is my bad, I shouldn't do `bash script.sh` in the first place